testing/parquet/parquet_partition.ecl

```diff
@@ -0,0 +1,46 @@
+//class=ParquetRegression
```

Comment:
How come this is using ParquetRegression instead of just parquet like the tests? It seems there are a few other cases as well.